Core/src/main/java/com/plotsquared/core/configuration/caption/load/CaptionLoader.java

```diff
@@ -93,7 +116,7 @@ private CaptionLoader() {
      * @see Files#list(Path)
      * @see #loadSingle(Path)
      */
-    @Nonnull public static CaptionMap loadAll(@Nonnull final Path directory) throws IOException {
+    public @NonNull CaptionMap loadAll(final @NonNull Path directory) throws IOException {
```

Comment:
why.. D:

Reply:
Don't question it :eyes: We're going to switch to checker-qual anyway as those annotations are actually following the new Java standard, so they'll all get moved around :P

Reply:
I mean it does make sense tbf, it's the "CaptionMap" that's not gonna be null, not "public". Just looks fugly when the rest of P2 doesn't do it that way